test/e2e/codeServer.test.ts

```diff
@@ -1,8 +1,45 @@
+import * as cp from "child_process"
 import { promises as fs } from "fs"
+import * as os from "os"
 import * as path from "path"
+import * as util from "util"
 import { describe, test, expect } from "./baseFixture"
+import { CodeServer } from "./models/CodeServer"
+
+describe("code-server", true, [], {}, () => {
+  // TODO@asher: Generalize this?  Could be nice if we were to ever need
+  // multiple migration tests in other suites.
+  const instances = new Map<string, CodeServer>()
+  test.afterAll(async () => {
+    const procs = Array.from(instances.values())
+    instances.clear()
+    await Promise.all(procs.map((cs) => cs.close()))
+  })
+
+  /**
+   * Spawn a specific version of code-server using the install script.
```

Comment:
Woaaah! You're using the install script? I love that this will mimic how users use code-server 👏